sdk/python/kfp_tekton/compiler/compiler.py

```diff
@@ -1394,13 +1388,10 @@ def get_when_task(input_task_when, depended_conditions):
 
     # add workflow level timeout to pipeline run
     if not TEKTON_GLOBAL_DEFAULT_TIMEOUT or pipeline.conf.timeout:
```

Comment:
looks like we don't need `TEKTON_GLOBAL_DEFAULT_TIMEOUT` anymore, should we also remove it?

Reply:
sure let me remove that

Reply:
done